lambda/templates/notifications_lambda.json.tpl

```diff
@@ -50,6 +50,23 @@
         "arn:aws:s3:::tdr-consignment-export-judgment-${environment}",
```

Comment:
There is an existing policy document that contains all the permissions for TRE separately, and this is where these permissions should go: https://github.com/nationalarchives/tdr-terraform-modules/blob/master/lambda/templates/notifications_transform_engine_lambda.json.tpl

Statement at line 28 should be updated to allow publishing to the Event Bus SNS topic: https://github.com/nationalarchives/tdr-terraform-modules/blob/master/lambda/templates/notifications_transform_engine_lambda.json.tpl#L28

Statement at line 34 should be updated to allow KMS key actions on the Event Bus KMS key: https://github.com/nationalarchives/tdr-terraform-modules/blob/master/lambda/templates/notifications_transform_engine_lambda.json.tpl#L34